course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -90,11 +90,103 @@ describe('Unit testing PeopleController', function() {
         it('won\'t bother waiting for profile lookups or POSTing new members ' +
             'if course membership lookup fails');
     });
+
     describe('addNewMember', function() {
-        it('won\'t add person if they already have a course enrollment');
-        it('won\'t add if they could not be found via /people lookup');
-        it('won\'t add if multiple profiles returned by /people lookup');
-        it('adds person if not in course and single profile available');
+
+        beforeEach(function() {
+            controller = $controller('PeopleController', {$scope: scope });
+            this.members = {};
```

Comment:
Nice use of `this`! So `this` refers to the context of the test? Why use `this` instead of `var`, just to indicate that the variables are relevant only to/within this `describe()` block and its children (not global, and not local to a specific `it()` block)?